### PR TITLE
Added new tests, added new method for uiMethods.helper.js following Y…

### PR DESCRIPTION
**Pull request by JuliaEde**
…ulia Har steps and copy pasted new method for ProfileEdit.page.js, added new element for ProfileEdit.page.js

### CODE REVIEW

File: test/helpers/uiMethods.helper.js [hunk 1]
```diff
@@ -18,9 +17,20 @@ async function randomString(length){
     }
     return result;
 }
+async function clearAndFillField(element, value){
+    while(await element.getValue() !== '') {
+        await element.doubleClick();
+        element.keys("Delete");
+    }
+    await element.click();
+    let result = await element.setValue(value);
```
Comment (valdisone) [inline test/helpers/uiMethods.helper.js:26]: it's not necessary to create variable, it's used only one time

File: test/ui/specs/user/profileEditFirstNameBoundary.spec.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        await Profile.btnEdit.click();
+    });
+
+    it('Should Show Error if First Name Length is 0 Characters', async () => {
+        await ProfileEdit.fillAndSave('');
+        await expect(ProfileEdit.pageTitle).toHaveText('user');
+    });
+
+    it('Should Accept Valid Length 1 Character', async () => {
+        let length1Name = 'J';
```
Comment (valdisone) [inline test/ui/specs/user/profileEditFirstNameBoundary.spec.js:30]: you can use the randomString() method to generate the string (as yuliahar mentioned), it was created but not yet used

File: test/helpers/uiMethods.helper.js (hunk 1 above)
Comment (JuliaEde) [reply test/helpers/uiMethods.helper.js:26]: Thank you for the update, should I remove return result? Then what will the function return?

Comment (JuliaEde) [reply test/helpers/uiMethods.helper.js:26]: return value?

File: test/ui/pageobjects/user/ProfileEdit.page.js [hunk 3]
```diff
@@ -29,39 +29,28 @@ class ProfileEditPage extends Page {
...
+
+    async fillAndSave(firstName, lastName, jobTitle, linkToImage, about) {
+        if (firstName !== undefined) {
+            await clearAndFillField(await this.inputFirstName, firstName);
         }
-        if (lastName !== ''){
-            await clearInputValue(await this.inputLastName);
-            await this.inputLastName.setValue(lastName);
+        if (lastName !== undefined) {
+            await clearAndFillField(await this.inputLastName, lastName);
```
Comment (valdisone) [inline test/ui/pageobjects/user/ProfileEdit.page.js:41]: extra line

File: test/helpers/uiMethods.helper.js (hunk 1 above)
Comment (valdisone) [reply test/helpers/uiMethods.helper.js:26]: `return await element.setValue(value);`

Comment (JuliaEde) [reply test/helpers/uiMethods.helper.js:26]: ох, вот так, окей, спасибо, я раньше не писала методы, довольно сложно, но интересно!

File: test/ui/specs/user/profileEditFirstNameBoundary.spec.js (hunk 2 above)
Comment (JuliaEde) [reply test/ui/specs/user/profileEditFirstNameBoundary.spec.js:30]: I tried but it failed, I don't know how to use it. It worked until the last step and when I execute expect it failed always.

File: test/ui/pageobjects/user/ProfileEdit.page.js (hunk 3 above)
Comment (JuliaEde) [reply test/ui/pageobjects/user/ProfileEdit.page.js:41]: if should start on the same line where the previous if finished? like so: } if () ?

Comment (JuliaEde) [reply test/ui/pageobjects/user/ProfileEdit.page.js:40]: I removed extra lines

File: test/helpers/uiMethods.helper.js (hunk 1 above)
Comment (JuliaEde) [reply test/helpers/uiMethods.helper.js]: Rewrote the method, please check

File: test/ui/specs/user/profileEditFirstNameBoundary.spec.js (hunk 2 above)
Comment (yuliahar) [reply test/ui/specs/user/profileEditFirstNameBoundary.spec.js:30]: Just remind, firstly import method
`const { randomString } = require('../../../helpers/uiMethods.helper');`
then you can 
`let length16Name = await randomString(16);`

**Review by yuliahar — CHANGES_REQUESTED** [review]

File: test/ui/specs/user/profileEditFirstNameBoundary.spec.js (hunk 2 above)
Comment (JuliaEde) [reply test/ui/specs/user/profileEditFirstNameBoundary.spec.js:30]: Yes, I had problems with using this method yesterday and Maria told me to use simple variables for now

Comment (yuliahar) [reply test/ui/specs/user/profileEditFirstNameBoundary.spec.js:30]: Please try to use this method another shot. If it continues to fail, please remind me to demonstrate how to apply this method during our meeting.

Comment (JuliaEde) [reply test/ui/specs/user/profileEditFirstNameBoundary.spec.js:30]: Thank you Yulia, I will try to use it, once you will approve the merge. I am confused about how to edit this file now, so many files have changed. I removed the element title from the Profile Edit page and xit -ed the test that was using it once, will fix it after a merge, thank you! Let me know if there's other things I have to edit in this PR

File: test/ui/pageobjects/user/ProfileEdit.page.js [hunk 4]
```diff
@@ -29,43 +29,28 @@ class ProfileEditPage extends Page {
...
-            await clearInputValue(await this.inputJobTitle);
-            await this.inputJobTitle.setValue(jobTitle);
-        }
-        await this.btnSave.click();
-    }
-    async fillAndSaveLinkToImage(linkToImage) {
-        await clearInputValue(await this.inputExternalLinkToProfileImage);
-        await this.inputExternalLinkToProfileImage.setValue(linkToImage);
-        await this.btnSave.click();
+    get pageTitle() {
```
[the remark this answers is not in the document]
Comment (JuliaEde) [reply test/ui/pageobjects/user/ProfileEdit.page.js]: Done, removed, xit -ed the it that was using this element, will change to navbar after merge

File: test/helpers/uiMethods.helper.js [hunk 5]
```diff
@@ -18,9 +17,18 @@ async function randomString(length){
     }
     return result;
 }
+async function clearAndFillField(element, value){
+    while(await element.getValue() !== '') {
```
[the remark this answers is not in the document]
Comment (JuliaEde) [reply test/helpers/uiMethods.helper.js]: Done

File: test/ui/specs/user/profileEditFirstNameBoundary.spec.js (hunk 2 above)
Comment (yuliahar) [reply test/ui/specs/user/profileEditFirstNameBoundary.spec.js:30]: ok

File: test/helpers/uiMethods.helper.js [hunk 6]
```diff
@@ -18,9 +17,15 @@ async function randomString(length){
     }
     return result;
 }
+async function clearAndFillField(element, value){
+    await clearInputValue(element);
+    await element.click();
```
Comment (yuliahar) [inline test/helpers/uiMethods.helper.js:22]: Please delete this line and try this method without await element.click.

File: test/ui/pageobjects/user/ProfileEdit.page.js (hunk 3 above)
Comment (yuliahar) [reply test/ui/pageobjects/user/ProfileEdit.page.js:37]: Please return lines back. There are different if-s, therefore they should start from a new line.

**Review by yuliahar — CHANGES_REQUESTED** [review]

File: test/helpers/uiMethods.helper.js (hunk 6 above)
Comment (JuliaEde) [reply test/helpers/uiMethods.helper.js]: Done, deleted

File: test/ui/pageobjects/user/ProfileEdit.page.js (hunk 3 above)
Comment (JuliaEde) [reply test/ui/pageobjects/user/ProfileEdit.page.js:38]: Done, returned them back

**Review by yuliahar — APPROVED** [review]

**Pull request merged**